package.json

```diff
@@ -65,11 +65,13 @@
   "devDependencies": {
     "@aptos-labs/aptos-cli": "^0.1.2",
     "@babel/traverse": "^7.23.6",
+    "bignumber.js": "^9.1.2",
     "@graphql-codegen/cli": "^5.0.0",
     "@graphql-codegen/import-types-preset": "^3.0.0",
     "@graphql-codegen/typescript": "^4.0.1",
     "@graphql-codegen/typescript-graphql-request": "^6.0.1",
     "@graphql-codegen/typescript-operations": "^4.0.1",
+    "@irys/sdk": "0.1.15",
```

Comment:
Irys is already at `0.2.1` - might worth to update?

Reply:
in the past ran into version mismatch issue. wanna get the functionality works first and then do version update in a separate PR.